pkg/mirrorregistries/generator.go

```diff
@@ -69,6 +70,18 @@ func (m *mirrorRegistriesConfigBuilder) GetMirrorRegistries() ([]byte, error) {
 	return os.ReadFile(m.MirrorRegistriesConfigPath)
 }
 
+// Determines whether or not this installation of assisted is using mirror registries
+func (m *mirrorRegistriesConfigBuilder) HasMirrorRegistries() (bool, error) {
+	hasMirrorRegistries := false
+	if _, err := m.GetMirrorRegistries(); err != nil {
+		if !os.IsNotExist(err) {
+			return false, err
+		}
+		hasMirrorRegistries = true
+	}
+	return hasMirrorRegistries, nil
```

Comment:
How it becomes true in case  m.GetMirrorRegistries() returns content?

Reply:
I got the order of my true/false mixed up here, good spot!